docs/cmdline-opts/page-footer

```diff
@@ -113,6 +113,8 @@ Makes it the equivalent of --socks5-hostname
 There are a bunch of different error codes and their corresponding error
 messages that may appear under error conditions. At the time of this writing,
 the exit codes are:
+.IP 0
+Success. The operation proceeded succesfully according to the instructions.
```

Comment:
s/succesfully/successfully. Also, it didn't just proceed successfully it _completed_ successfully.